utils/auth_jwt.ts

```diff
@@ -7,9 +7,16 @@ import { pipe } from "fp-ts/lib/function";
 import { NonEmptyString } from "@pagopa/ts-commons/lib/strings";
 
 import { Second } from "@pagopa/ts-commons/lib/units";
+import { IRequestMiddleware } from "@pagopa/io-functions-commons/dist/src/utils/request_middleware";
+import {
+  IResponseErrorValidation,
+  ResponseErrorValidation
+} from "@pagopa/ts-commons/lib/responses";
+import { readableReportSimplified } from "@pagopa/ts-commons/lib/reporters";
 import { AssertionRef } from "../generated/definitions/internal/AssertionRef";
 import { OperationId } from "../generated/definitions/internal/OperationId";
 
+import { LollipopAuthBearer } from "../generated/definitions/external/LollipopAuthBearer";
```

Comment:
The name of this type should be generic, not only for lollipop.
es `JWTAuthBearer`